### PR TITLE
Building with bdist_rpm

### PR DESCRIPTION
**Pull request by itkovian**

### CODE REVIEW

**Comment by stdweird** [comment]
what are you trying to achieve here? aren't there enough standard packages for sqlalchemy?

**Comment by JensTimmerman** [comment]
can't you send this pull request to https://github.com/zzzeek/sqlalchemy ? So we don't need to maintain this?

File: setup.py [hunk 1]
```diff
@@ -11,6 +11,26 @@
...
+
+def remove_bdist_rpm_source_file():
+    """List of files to remove from the (source) RPM."""
+    return ['lib/vsc/__init__.py']
+
+
+shared_setup.remove_extra_bdist_rpm_files = remove_bdist_rpm_source_file
+shared_setup.SHARED_TARGET.update({
+    'url': 'https://github.ugent.be/hpcugent/vsc-config',
+    'download_url': 'https://github.ugent.be/hpcugent/vsc-config'
```
Comment (JensTimmerman) [inline setup.py]: how does this end up here?

File: setup.py [hunk 2]
```diff
@@ -11,6 +11,26 @@
...
+    import vsc.install.shared_setup as shared_setup
+    from vsc.install.shared_setup import ag
+except ImportError:
+    print "vsc.install could not be found, make sure a recent vsc-base is installed"
+    print "you might want to try 'easy_install [--user] https://github.com/hpcugent/vsc-base/archive/master.tar.gz'"
+
+
+def remove_bdist_rpm_source_file():
+    """List of files to remove from the (source) RPM."""
+    return ['lib/vsc/__init__.py']
```
Comment (JensTimmerman) [inline setup.py]: this file isn't in this repo, I see no reason to use vsc things here?

**Comment by itkovian** [comment]
@stdweird No el6 packages that are bigger than 1.0 afaik.

**Comment by JensTimmerman** [comment]
you can create rpm's from repo's on github with e.g.

`python clusterbuildrpm.py -r pub -p sqlalchemy -f zzzeek -b rel_1_0_8 --user=jgtimmer --os=sl6`

there was an issue in clusterbuildrpm-server which made this fail for sqalchemy, but I fixed this in:
https://github.ugent.be/hpcugent/clusterbuildrpm-server/pull/16

**Comment by JensTimmerman** [comment]
I suggest we close this pr and delete this fork?

**Comment by itkovian** [comment]
Yes.

**Pull request closed without merging**